src/lib/customers/20minuten-de.json

```diff
@@ -0,0 +1,54 @@
+{
+	"title": "bespinian unterstützt 20 Minuten bei der Modernisierung der digitalen Produkte",
+	"logo": {
+		"file": "tx-group.webp",
+		"alt": "TX Group"
+	},
+	"customer": "<a href=\"https://www.20min.ch/\" target=\"_blank\">20 Minuten</a> ist die reichweitenstärkste Medienmarke der Schweiz. Sie gehört wie auch das Medienhaus Tamedia und die Werbevermarkterin Goldbach zur TX Group. 20 Minuten zeichnet sich durch ihre Reichweite in allen Sprachregionen und dem Storymix aus News, Unterhaltung und Inspiration aus und war/ist für bespinian ein wichtiger Partner in der Zusammenarbeit bezüglich der Migration der digitalen Produkte auf eine neue Infrastruktur, und anderen Projekten im Rahmen von OAuth2, Cloudifizierung, Containerisierung und Modernisierung von Software.",
```

Comment:
Replace `und dem Storymix aus News, Unterhaltung und Inspiration aus und war/ist für bespinian ein wichtiger Partner in der Zusammenarbeit` by `und dem Storymix aus News, Unterhaltung und Inspiration aus. Das 20 Minuten Team war und ist für bespinian ein wichtiger Partner in der Zusammenarbeit ...`